content/_includes/CLI/AccountGroupDeleteExample.md

```diff
@@ -0,0 +1,17 @@
+---
+---
+
+```
+account group delete id=<GID>
+```
+
+Where `<GID>` is the group identification number.
+
+The command returns a blank line.
+To confirm the group is deleted, use `account group query`, `account group get_group_obj`, or navigate to [**Credentials > Local Groups**]({{< relref "managelocalgroups.md" >}}) in the SCALE Web UI.
```

Comment:
Move this out of the command example and up to the details about using the command after the first expand for the command section, see the comment in the CLIgroup.md article review.